dialog-el.html

```diff
@@ -6,11 +6,19 @@
     :host {
       position: absolute;
       display: inline-block;
+      transform: translate(0,0);
     }
     :host([hidden]) {
       display: none !important;
     }
-    
+    :host([attach]) {
```

Comment:
Sorry, yes, I do need to document it. `arrow-direction` is much easier to understand, so I will go ahead and change that.